app/assets/javascripts/miq_list_grid.js

```diff
@@ -41,12 +41,12 @@ function miqOrderService(id) {
 }
 
 // Handle checkbox
-function miqGridOnCheck(elem, button_div) {
+function miqGridOnCheck(elem, button_div, grid) {
```

Comment:
@romanblanco why do we need third parameter `grid` if we are always passing in same value.